docker/docker-compose-without-neo4j.yml

```diff
@@ -66,8 +66,6 @@ services:
       dockerfile: docker/datahub-upgrade/Dockerfile
     env_file: datahub-upgrade/env/docker-without-neo4j.env
     depends_on:
-      mysql-setup:
```

Comment:
We only mention mysql in the `.override` files. This `depends_on` gets added by `docker-compose-without-neo4j.override.yml`